datameta_client/shortcuts.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2021 Universität Tübingen, Germany
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import typer
+from typing import Optional, List
+
+from .config import get_config
+from .utils import get_list_or_dict_from
+from .printing import info, success, result, error
+from . import metadatasets, files, submissions
+
+app = typer.Typer()
+
+
+@app.command()
+def stage_and_submit(
+    metadatasets_json, # a single or a list of records dicts,
+                       # or the equivalent in a json string or file
+    files_dir:Optional[str],
+    label:Optional[str] = None,
+    url:Optional[str] = None, 
+    token:Optional[str] = None,
+    quiet:bool = False,
+) -> bool:
+    info(
+        "This will stage and directly submit metadatasets and files in a single run.",
+        quiet
+    )
+
+    # parse metadatasets_json:
+    info("Parsing metadata", quiet)
+    msets = get_list_or_dict_from(metadatasets_json)
+    msets = metadatasets if isinstance(msets, list) else [msets]
```

Comment:
This crashed with a list
```suggestion
    msets = msets if isinstance(msets, list) else [msets]
```